File: backEnd/src/main/java/com/quiz/ourClass/domain/challenge/entity/Report.java
package com.quiz.ourClass.domain.challenge.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.OneToOne;
import java.time.LocalDateTime;

@Entity
public class Report {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    long id;
    @OneToOne(fetch = FetchType.LAZY)
    ChallengeGroup challengeGroup;
    String content;
    String file;
    LocalDateTime createTime;
    boolean acceptStatus;
}

File: android/core/model/src/main/java/com/sixkids/model/Report.kt
package com.sixkids.model

import java.time.LocalDateTime

data class Report(
    val id: Long = 0,
    val group: Group = Group(),
    val startTime: LocalDateTime = LocalDateTime.now(),
    val endTime: LocalDateTime = LocalDateTime.now(),
    val file : String = "",
    val content: String = "",
    val acceptStatus: AcceptStatus = AcceptStatus.BEFORE,
)
